### PR TITLE
Added Suggested Actions info

### PR DESCRIPTION
**Pull request by amberben**
Add a new tab for suggested actions, added details on each new section added to tab

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/explorer.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Below the header of Cloud SIEM and Cloud Workload Security signals are tabs with
 - `Rule Details` displays rule details, such as the text configured in the detection rule to help the person reviewing the signal understand the purpose of the signal and how to respond. The users can also pivot into rule modification, such as modifying suppression queries for the rule.
 - `Logs` includes a visualization and list of log samples to provide context on why the signal triggered. Click on any of the samples in the table to see the full log.
 - `Related Signals` are displayed as a timeline of other signals which contain the same group by values to assist with triaging the signal.
+- `Suggested Actions (beta)` provides investigation queries, related dashboards and pivots into cloud provider consoles based on Security Signal characteristics that guide investigations and provide insights into needed resolution.
```
Comment (maycmlee) [inline content/en/security_platform/explorer.md:56]:
```suggestion
- `Suggested Actions (beta)` provides investigation queries, related dashboards and links to cloud provider consoles based on Security Signal characteristics that guide investigations and provide insights into needed resolution.
```

**Review by maycmlee — COMMENTED** [review]
Just a small edit!

**Comment by maycmlee** [comment]
Thanks @amberben for the addition! Let me know if the edits work for you.

**Comment by amberben** [comment]
The edits work for me -- thank you @maycmlee !

**Pull request merged**